lib/availability_status_listener.rb

```diff
@@ -62,8 +62,8 @@ def process_event(event)
     Rails.logger.error(["Something is wrong when processing Kafka message: ", e.message, *e.backtrace].join($RS))
   end
 
-  def raise_event(event, payload)
-    Sources::Api::Events.raise_event(event, payload)
+  def raise_event(event, payload, headers)
```

Comment:
Is there a chance this should be `headers=nil` by default?

Reply:
I debated that, but the sources event class already handles it for us:
https://github.com/lindgrenj6/sources-api/blob/8e904b4371e2d716061155cc30978385feae3a2c/lib/sources/api/events.rb#L4

Reply:
@lindgrenj6 makes sense.  Sounds good.